core/Makefile

```diff
@@ -78,6 +78,9 @@ emu: ## run emulator
 test: ## run unit tests
 	cd tests ; ./run_tests.sh $(TESTOPTS)
 
+test_rust: ## run rs unit tests
+	cd embed/rust ; cargo test -- --test-threads=1
```

Comment:
btw this needs to be changed to `cargo test --features="test ui_debug" -- --test-threads=1` when the UI branch is merged